Packs/IBMResilientSystems/Scripts/IbmConvertTasksToTable/IbmConvertTasksToTable.yml

```diff
@@ -0,0 +1,16 @@
+commonfields:
+  id: IbmConvertTasksToTable
+  version: -1
+comment: This script is used to convert IBM QRadar SOAR notes to a markdown table.
```

Comment:
Should this be tasks?
This script is used to convert IBM QRadar SOAR tasks to a markdown table.